shell/components/fleet/FleetBundles.vue

```diff
@@ -127,7 +127,8 @@ export default {
             class="text-warning"
           >
             {{ row.status.summary.ready }}/{{ row.status.summary.desiredReady }}</span>
-          <span v-else-if="row.status">{{ row.status.summary.desiredReady }}</span>
+          <span v-else-if="row.status && row.status.summary">{{ row.status.summary.desiredReady }}</span>
```

Comment:
Does the syntax `row.status?.summary` work here?

Reply:
It never used to but now does. Could be a Vue 2.5 --> 3 change.

Will leave this one as is and update in future